app/models/state_file_nj_intake.rb

```diff
@@ -201,8 +202,5 @@ def disqualifying_eligibility_rules
 
   def validate_state_specific_w2_requirements(w2); end
 
-  def ask_for_signature_pin?
-    true
-  end
```

Comment:
[dust/pebble] does this default to false? Maybe we could keep the method but have it explicitly return false, since that is explicit logic that we want to define for NJ rather than relying on an external default